src/components/Independent/Independent.js

```diff
@@ -25,12 +25,31 @@ const Independent = ({
   status_msg,
   isPinned,
   removeTikTok,
-  addPin,
-  removePin
+  togglePin
   }) => {
 
-  const togglePin = () => {
-    isPinned ? removePin(data_video_id) : addPin(data_video_id)
+  const determineErrorRender = () => {
+    if (status_msg) {
+      return (
+        <CardContent>
+          <p>Not able to load this tiktok, please check your path url</p>
```

Comment:
Love the clear and concise error message but I think it is getting too technical for the average tiktok user. Would they know what the path url is/should be and know what to do to fix it? 

Might be more non-tech friendly to have them return to the home page and try again or just try a different video.